packages/elements/src/components/table/__styles__/index.tsx

```diff
@@ -140,6 +320,8 @@ export const ElTableCell = styled.div`
     margin-right: 0;
   }
 
+  ${cellOrders}
+
   ${ElIcon} {
```

Comment:
I don't think this works... I appreciate, not your add but worth checking

Reply:
you mean ElIcon? I'll check it out

Reply:
<img width="112" alt="Screenshot 2021-06-30 at 16 22 20" src="https://user-images.githubusercontent.com/11574250/123987782-61ef8480-d9bf-11eb-88d3-10cd292663da.png">
this adds a margin right to the icon here